docs/docs/cmd/spo/tenant/tenant-site-membership-list.mdx

```diff
@@ -0,0 +1,93 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo tenant site membership list
+
+Retrieve information about default site groups' membership
+
+## Usage
+
+```sh
+m365 spo tenant site membership list [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --siteUrl <siteUrl>`
+: The URL of the site.
+
+`-r, --role [role]`
+: Filter the results to include only users with the specified roles: 'Owner', 'Member', or 'Visitor'.
+
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To execute this command successfully, ensure you are logged in to the CLI for M365 with an account that has SharePoint admin permissions. For other scenarios, refer to the `spo web get --withGroups` and `spo group member list` commands.
```

Comment:
Let's just display: `To use this command, you must be a Global or SharePoint administrator.` to make it consistent with other commands.

Reply:
Done